packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -432,15 +452,17 @@ export class ObjectSetListenerWebsocket {
     );
     const osdkObjectsWithReferenceUpdates = await Promise.all(
       referenceUpdates.map(async (o) => {
-        const osdkObjectArray = await convertWireToOsdkObjects(
+        const osdkObjectArray = await this.#client.objectFactory(
```

Comment:
I'm using the old interface API's since the our new implementations require and interface to object type properties map. I could manually mock this, but I think it would be best to talk through this @ssanjay1 and just unblock for right now